azteco.md

```diff
@@ -0,0 +1,196 @@
+
+The process is simple - users buy a voucher with a unique code from a participating location or online and then redeem this code on the Azteco website. The equivalent Bitcoin value is transferred directly to the user's wallet upon redemption. 
+
+## Preparation
+
+### Bitcoin Wallet 
+
+You need a Bitcoin wallet where you will receive the funds. In this guide, we will leverage a new feature from [Envoy](https://foundationdevices.com/envoy/) called "Azteco Voucher Redemption". This feature lets you scan Azteco's QR code and directly transfer sats to your Passport or Envoy wallet. Otherwise, you need a receiving Bitcoin address from any standard wallet.
+
+If you need a Bitcoin wallet, consider the following options (Avoid Azteco's recommendations):
+
+**On-chain wallets:**
+- [Envoy](https://foundationdevices.com/envoy/) – Android/iOS
+- [Samourai Wallet](https://samouraiwallet.com/) – Android Only
+- [Sparrow Wallet](https://sparrowwallet.com/) – Desktop
+
+**Lightning wallets:**
+- [Zeus](https://zeusln.app/) – Android/iOS
+- [Phoenix](https://phoenix.acinq.co/) – Android/iOS
+- [Blixt](https://blixtwallet.github.io/) – Android/iOS
+
+## Buy Vouchers
+
+*Note that Azteco offers lightning and on-chain vouchers. In this guide, we will buy an on-chain voucher.*
+
+### Physical Vendors
+The **best** way to acquire your Azteco voucher is through its network of vendors. Check Azteco [official resellers' map](https://azte.co/find_a_vendor) and see if you can trade cash for sats around you. This is the most private way to acquire vouchers as you can pay with cash and bypass request for identification.
+
+### Order online from Azteco
+Azteco offers a hybrid online and in-person service, available in a variety of countries, that allows you to purchase Bitcoin vouchers. The service provides numerous payment methods, including cash, bank transfer, and credit card. Azteco has [cash-in locations](https://azte.co/pay-in-person) across several countries including Brazil, Colombia, Mexico, Peru, Chile, Costa Rica, South Africa, Ghana, Uganda, Zambia, and Ireland.
+
+Start by navigating to the [Pay-in-person](https://azte.co/pay-in-person) page. This page will provide you with all the necessary instructions to order your voucher online. You need to fill-up a form in order to get a bar code. Once you have this bar code you can go to any of the Azteco-affiliated physical locations. This physical vendor will scan your barcode and collect your payment. The vendor will communicate this transaction to Azteco. Once payment is confirmed, Azteco emails you the voucher.
+
+### Online resellers
+If there aren't any Azteco vendors in your area, you can purchase a voucher online from [approved retailers](https://help.azte.co/article/8-buy-a-voucher-approved-retailers) like G2A or Kinguin. However, it's important to remember that online purchases will require an email address and potentially less private payment methods compared to cash transactions.
```

Comment:
help URL also not loading for me!?

Reply:
Weird, they changed the number in /**8**-buy-a-voucher-approved-retailers. Thanks for checking!